README.md

```diff
@@ -77,3 +77,182 @@ This is a list of the required environment variables:
 #### MSSQL Server
 
 **SA_PASSWORD** - MSSQL Server "SA" user password
+
+## SQL Server Database Schema
+
+```mermaid
+
+erDiagram
+
+    Projects {
+        int Id PK
+        int DefaultBranchId FK
+        nvarchar(50) Name
+        int EngineEnum
+    }
+
+    Commits {
+        int Id PK
+        int AuthorId FK
+        int ParentId FK"null"
+        nvarchar(200) Message 
+        datetime CreatedAt 
+        nvarchar(500) SnapshotUrl 
+    }
+
+    BranchCommits {
+        int Id PK
+        int CommitId FK
+        int BranchId FK
+        bit IsMerged 
+        bit IsHead 
+    }
+
+    Tags {
+        int Id PK
+        nvarchar(50) Name    
+    }
+
+    ProjectSettings {
+        int Id PK
+        int ProjectId FK
+        nvarchar(500) SettingsUrl
+    }
+
+    Branches {
+        int Id PK
+        int ProjectId FK
+        nvarchar(200) Name 
+        bit IsActive 
+    }
+
+    PullRequests {
+        int Id PK
+        int ProjectId FK
+        int AuthorId FK
+        int SourceBranchId FK
+        int TargetBranchId FK
+        nvarchar(100) Title 
+        int StatusEnum 
+        bit IsReviewed 
+        datetime CreatedAt 
+        datetime UpdatedAt 
+    }
+
+    PullRequestReviewers {
+        int Id PK
+        int PullRequestId FK
+        int UserId FK
+    }
+
+    Comments {
+        int Id PK
+        int PullRequestId FK
+        int AuthorId FK
+        nvarchar(500) Content 
+        datetime CreatedAt 
+        datetime UpdatedAt 
+    }
+
+    UserGroups {
+        int Id PK
+        int GroupId FK
+        int UserId FK
+    }
+
+    Groups {
+        int Id PK
+        nvarchar(50) Name
+        bit IsDefault
+    }
+
+    Users {
+        int Id PK
+        nvarchar(25) UserName
+        nvarchar(25) FirstName
+        nvarchar(25) LastName
+        nvarchar(50) Email
+        nvarchar(25) Password
```

Comment:
it is bad to store user's password explicitly, so maybe we should hash it with salt. In this case User entity should have 'Salt' attribute of nvarchar type